commands/start.go

```diff
@@ -50,20 +61,41 @@ func NewKoolStart() *KoolStart {
 	defaultKoolService := newDefaultKoolService()
 	return &KoolStart{
 		*defaultKoolService,
-		&KoolStartFlags{false},
+		&KoolStartFlags{false, false},
 		checker.NewChecker(defaultKoolService.shell),
 		network.NewHandler(defaultKoolService.shell),
 		environment.NewEnvStorage(),
 		compose.NewDockerCompose("up", "--force-recreate"),
+		&KoolRebuild{
+			*newDefaultKoolService(),
+			compose.NewDockerCompose("pull"),
+			compose.NewDockerCompose("build", "--pull"),
```

Comment:
Isn't the pull operation being executed twice?

Reply:
@danielsuguimoto no! the first pull only happens for the images declared directly with `image:` on the docker-compose.yml file. While the --pull for the build will make that the build attempts pulling newer images for the base images being used on the docker files. We need both.